BUILD.bazel

```diff
@@ -60,157 +60,216 @@ filegroup(
-        "mock/dataArrayB_256x64.sv",
-        "mock/ebtb_128x40.sv",
-        "mock/l2_tlb_ram_0_512x46.sv",
-        "mock/regfile_128x64.sv",
-        "mock/regfile_128x65.sv",
-        "mock/tag_array_64x184.sv",
+    "mock/array_256x128.sv",
+    "mock/data_2048x8.sv",
+    "mock/dataArrayB_256x64.sv",
+    "mock/ebtb_128x40.sv",
+    "mock/l2_tlb_ram_0_512x46.sv",
+    "mock/regfile_128x64.sv",
+    "mock/regfile_128x65.sv",
+    "mock/tag_array_64x184.sv",
 ]
 
 all_source_files = sorted(glob(
-    include = ["rtl/*.sv", "rtl/*.v"],
-    exclude = ([f.replace("mock/", "rtl/") for f in mock_files] +
-    ["rtl/ClockSourceAtFreqMHz.v",
-    "rtl/SimJTAG.v",
-    "rtl/SimTSI.v"])
+    include = [
+        "rtl/*.sv",
+        "rtl/*.v",
+    ],
+    exclude = ([f.replace("mock/", "rtl/") for f in mock_files] + [
+        "rtl/ClockSourceAtFreqMHz.v",
+        "rtl/SimJTAG.v",
+        "rtl/SimTSI.v",
+    ]),
 ) + mock_files)
 
 boom_tile_small_srams = [
-        "tag_array_64x184",
-        "tag_array_64x168",
-        # "data_2048x2",
-        "table_256x48",
-        "table_128x52",
-        "table_128x44",
-        "btb_128x56",
-        "meta_128x120",
-        "lb_32x128",
-        "sdq_17x64",
-        "data_2048x8",
-        "mem_256x4"]
+    "tag_array_64x184",
+    "tag_array_64x168",
+    # "data_2048x2",
+    "table_256x48",
+    "table_128x52",
+    "table_128x44",
+    "btb_128x56",
+    "meta_128x120",
+    "lb_32x128",
+    "sdq_17x64",
+    "data_2048x8",
+    "mem_256x4",
+]
 
 boom_tile_rams = [
-        # "l2_tlb_ram_0_512x46",
-        "ebtb_128x40",
-        "array_256x128",
-        "dataArrayB_256x64"
-        #, "l2_tlb_ram_0_512x45"
-        ]
-
-digital_top_srams=[
-        # "cc_dir_1024x168",
-        # "data_data_40x128",
-        # "ghist_40x64",
-        "meta_40x240"]
-
-
-[build_openroad(
-        docker_image = orfs_version,
+    # "l2_tlb_ram_0_512x46",
+    "ebtb_128x40",
+    "array_256x128",
+    "dataArrayB_256x64",
+    # "l2_tlb_ram_0_512x45",
+]
+
+digital_top_srams = [
+    # "cc_dir_1024x168",
+    # "data_data_40x128",
+    # "ghist_40x64",
+    "meta_40x240",
+]
+
+[
+    orfs_flow(
         name = ram,
-        io_constraints=":io-sram",
-        verilog_files=[{'array_256x128':'mock',
-        'cc_dir_1024x168':'mock',
-        'data_2048x2':'mock',
-        'dataArrayB_256x64':'mock',
-        'data_data_40x128':'mock',
-        'ebtb_128x40':'mock',
-        'ghist_40x64':'mock',
-        'l2_tlb_ram_0_512x45':'mock',
-        'tag_array_64x184':'mock',
-        'data_2048x8':'mock'}.get(ram, "rtl") + "/" + ram + ".sv"],
-        sdc_constraints = ":constraints-sram",
-        stage_args={
-                'synth': ['SYNTH_MEMORY_MAX_BITS=16384'],
-                'floorplan': ['CORE_UTILIZATION=40',
-                        'CORE_ASPECT_RATIO=2'],
-                'place': ['PLACE_DENSITY=0.65',
-                'GPL_TIMING_DRIVEN=0'],
-                'cts': ['SKIP_CTS_REPAIR_TIMING=1',
```

Comment:
add SKIP...REPAIR_TIMING=0 placeholders until SKIP...REPAIR_TIMING=1 can be fixed. I don't want to loose the information on where I was doing this as it relates to performance problems in openroad that I want to track by using SKIP...REPAIR_TIMING=1